### PR TITLE
Adding OS APIPoller for Petro

### PR DESCRIPTION
**Pull request by grantoesterling**
## What's New
- OS API Poller for Petro National lists/sales (I already had an OS poller mocked, so this is a quick and dirty solution - will come back to this next week and replace with a more reusable solution)
- Additionally posts sales to the ab-x-pace channel

## Screenshots
Listings
Screen Shot 2022-06-21 at 3 51 20 PM
Sales
Screen Shot 2022-06-21 at 3 51 43 PM

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -246,3 +247,29 @@ new LooksRareAPIPollBot(
...
+new OpenseaAPIPollBot(
+  `https://api.opensea.io/api/v1/events?collection_slug=${paceSlug}&event_type=created`,
+  API_POLL_TIME_MS,
+  bot,
+  {
+    Accept: 'application/json',
+    'X-API-KEY': process.env.OPENSEA_API_KEY,
+  },
+  COLLAB_CONTRACTS.AB_X_PACE
+)
```
Comment (ryley-o) [inline index.js:275]: @grantoesterling you sure we want created?

**Review by ryley-o — APPROVED** [review]
LGTM! - one question about if we want to include `created` events (which I think are when new listings are created?)

File: index.js (hunk 1 above)
Comment (ryley-o) [reply index.js:275]: (I'm also not up to speed on artbot, so maybe we post created events now? 😆 )

Comment (grantoesterling) [reply index.js:275]: pompey says yes!

**Pull request merged**